File: Fresh/Fresh.Model/Notification.cs
namespace Fresh.Model
{
    public class Notification
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public DateTime CreatedAt { get; set; }
        public int CompanyId { get; set; }

        public CompanyInfo Companies { get; set; }
    }
}

File: Fresh/UI/fresh_app/src/app/model/Notification.ts
import { CompanyInfo } from "./CompanyInfo";

export class Notification {
    id: number;
    title: string;
    content: string;
    createdAt: Date;
    companyId: number;

    company: CompanyInfo;

    constructor(id: number,
        title: string,
        content: string,
        createdAt: Date,
        companyId: number,
        company: CompanyInfo) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.createdAt = createdAt;
        this.companyId = companyId;
        this.company = company;
    }

    static fromJson(json: any): Notification {
        return new Notification(
            json.id,
            json.title,
            json.content,
            json.createdAt,
            json.companyId,
            json.company
        );
    }
}